changelog/7799.bugfix.md

```diff
@@ -0,0 +1 @@
+Updated validator used by `rasa data validate` to verify that actions used in stories and rules are present in the domain and that form slots match domain slots.
```

Comment:
I'd argue that this is rather an `improvement` than a bugfix and should hence go into `main` instead of 2.6.x.

Reply:
Ok, I'll attempt rebase 😓